docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
+services:
+  web:
+    build: .
+    ports:
+      - "80:80"
```

Comment:
You might want to specify the network and depends on it just so the web service waits on the Nginx service to complete before it spins up.

You might wanna specify volumes as well for caching purposes.

Reply:
I am guessing it will be Ngnix depending on the application. I forgot but you might want to find that out.

Also, give these containers names. then it'll be easier when you debug then. `container_name` field.